Makefile

```diff
@@ -54,27 +54,14 @@ draw_deps:
 	go get github.com/RobotsAndPencils/goviz
 	@goviz -i github.com/tendermint/tendermint/cmd/tendermint -d 3 | dot -Tpng -o dependency-graph.png
 
-list_deps:
-	@go list -f '{{join .Deps "\n"}}' ./... | \
-		grep -v /vendor/ | sort | uniq | \
-		xargs go list -f '{{if not .Standard}}{{.ImportPath}}{{end}}'
-
-get_deps:
-	@echo "--> Running go get"
-	@go get -v -d $(PACKAGES)
-	@go list -f '{{join .TestImports "\n"}}' ./... | \
-		grep -v /vendor/ | sort | uniq | \
-		xargs go get -v -d
-
-update_deps:
-	@echo "--> Updating dependencies"
-	@go get -d -u ./...
-
 get_vendor_deps:
 	@hash glide 2>/dev/null || go get github.com/Masterminds/glide
 	@rm -rf vendor/
 	@echo "--> Running glide install"
-	$(GOPATH)/bin/glide install
+	@$(GOPATH)/bin/glide install
```

Comment:
wasn't greg asking for us to change this? Ie. what if glide is installed via pkg manager and not go ?

Reply:
no, he was asking to change `@hash glide 2>/dev/null` above to `@hash $(GOPATH)/bin/glide 2>/dev/null`. but I am not sure this is correct. my intention was if glide is already present (e.g. user did `brew install glide`) than we do not attempt to install it ourselves